.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,46 @@
+name: Deploy Production
+
+on:
+  workflow_dispatch:
+    inputs:
+      git_commit:
+        description: "Short Git commit hash to deploy to production"
+        required: true
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    environment: production
+    env:
+      AWS_DEFAULT_REGION: "us-east-1"
+      BATCH_ANALYSIS_JOB_NAME_REVISION: "dummy-test-pfb-analysis-run-job:1"
+      DOCKER_BUILDKIT: 1
+      ENVIRONMENT: "production"
```

Comment:
This is a key difference than the CI workflow, since the scripts used in the workflow has `ENVIRONMENT: "staging"` as default.